SimpleWeather/Model/WeatherCity.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  WeatherCity.swift
+//  SimpleWeather
+//
+//  Created by Alexander on 21.06.2020.
+//  Copyright © 2020 Alexander Team. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+class WeatherCity: Object {
+    
+    @objc dynamic var cityName: String
```

Comment:
модель называется city, зачем писать какой это нейм?
```suggestion
    @objc dynamic var name: String
```